### PR TITLE
Small Additions

### PR DESCRIPTION
**Pull request by Faf4a**
## Added
- `$getObjectValues[object;sep?]`
- `$getObjectKeys[object;sep?]`
- `$objectExists[object]`
- `$deleteObject[object]`
- `$cloneObject[object;cloned object name]`

## Changes
- `$createObject[objectname;json]`
  - Ability to create multiple objects.
- `$getObjectProperty[objectname;property]`
- Renamed `$addObjectProperty` to `$setObjectProperty` and changed usage to `[objectname, propertyname, propertyvalue]`
- All object functions now work "together", as well you're now able to create multiple objects.
- made `errorMessage` support parser
- fixed `$addTimestamp`
- fixed `$getLeaderboardInfo`

### CODE REVIEW

**Comment by Faf4a** [comment]
## Added
- `$getObjectValues[object]`
- `$getObjectKeys[object]`
- `$objectExists[object]`
- `$deleteObject[object]`
- `$cloneObject[object;cloned object name]`

## Changes
- `$createObject[objectname;json]`
 - Ability to create multiple objects.
- `$getObjectProperty[objectname;property]`
- Renamed `$addObjectProperty` to `$setObjectProperty` and changed usage to `[objectname, propertyname, propertyvalue]`
- All object functions now work "together", as well you're now able to create multiple objects.

**Comment by Faf4a** [comment]
Fixed `$addTimestamp` to actually be able to add timestamps, for example: `$addTimestamp[1;1689706824079ms]` or `$addTimestamp[1689706824079ms]`.

**Comment by Faf4a** [comment]
Made `$getLeaderboardInfo` support only `aoi.db` for now as it's the main database + dbdjs.db is unstable, aoimongo died ages ago no idea about the other one.

**Review by USERSATOSHI — APPROVED** [review]
LGTM

File: src/core/interpreter.js
Comment (Leref) [inline src/core/interpreter.js:1]: Revert this as interpreter will change.

**Review by Leref — CHANGES_REQUESTED** [review]

**Pull request merged**